runtime/opt/taupage/init.d/02-prepare-td-agent.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
#!/usr/bin/env bash

Reply:
Why? We use `#!/bin/bash` for most scripts.